lib/rspec/openapi.rb

```diff
@@ -2,13 +2,29 @@
 require 'rspec/openapi/hooks' if ENV['OPENAPI']
 
 module RSpec::OpenAPI
-  @path = 'doc/openapi.yaml'
+  @application_version = '1.0.0'
   @comment = nil
-  @enable_example = true
+  @description = nil
   @description_builder = -> (example) { example.description }
-  @application_version = '1.0.0'
+  @enable_example = true
+  @path = 'doc/openapi.yaml'
+  @summary_builder = -> (example) { example.metadata[:summary] }
+  @tags = []
+  @title = 'app'
 
   class << self
-    attr_accessor :path, :comment, :enable_example, :description_builder, :application_version
+    ACCESSORS = %i(
+      application_version
+      comment
+      description
+      description_builder
+      enable_example
+      path
+      summary_builder
+      tags
+      title
+    )
+
+    attr_accessor(*ACCESSORS)
```

Comment:
I prefer not to expose an unnecessary constant.

```suggestion
    attr_accessor(
      :application_version,
      :comment,
      :description,
      :description_builder,
      :enable_example,
      :path,
      :summary_builder,
      :tags,
      :title,
    )
```

Reply:
The cool thing about the constant is that it can be tested, and it's easier to keep alphabetically sorted. But I will change that before submitting it for final approval 👍🏼